descope/sdk/auth.go

```diff
@@ -313,6 +313,12 @@ type Authentication interface {
 	// the specified roles for a specific tenant.
 	ValidateTenantRoles(token *descope.Token, tenant string, roles []string) bool
 
+	// SelectTenantWithRequest - Use to add to the jwt a dedicated claim to mark on which tenant this user authenticated now
+	SelectTenantWithRequest(tenantID string, request *http.Request, w http.ResponseWriter) (*descope.AuthenticationInfo, error)
+
+	// SelectTenantWithToken - Use to add to the jwt a dedicated claim to mark on which tenant this user authenticated now
```

Comment:
with respect to the above

```suggestion
	// SelectTenantWithToken - Adds a dedicated claim to the JWTs to indicate the tenant on which the user is currently authenticated
```

Reply:
done